tests/NexySlackBundleTest.php

```diff
@@ -33,6 +33,9 @@ protected function setUp(): void
         $this->bundle = new NexySlackBundle();
     }
 
+    /**
+     * @doesNotPerformAssertions
```

Comment:
PHPUnit marks tests that do not perform any assertions as risky, this annotation silences that.

Reply:
Nice catch! :+1: